infra/gravity/cluster_install.go

```diff
@@ -232,7 +232,7 @@ func (c *TestContext) upgrade(master Gravity, numNodes int) error {
 
 // ExecScript will run and execute a script on all nodes
 func (c *TestContext) ExecScript(nodes []Gravity, scriptUrl string, args []string) error {
-	ctx, cancel := context.WithTimeout(c.ctx, c.timeouts.Status)
+	ctx, cancel := context.WithTimeout(c.ctx, c.timeouts.ExecScript)
```

Comment:
As mentioned below, this timeout should be specified by the end user who specifies the exec script.  Needs a followup ticket.

Reply:
Followup PR: https://github.com/gravitational/robotest/pull/235